playwright-e2e/tests/rgp/kit-upload-blood-and-rna.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { test, expect, Page } from '@playwright/test';
+import { login } from 'authentication/auth-dsm';
+import HomePage from 'pages/dsm/home-page';
+import { Navigation } from 'lib/component/dsm/navigation/navigation';
+import { WelcomePage } from 'pages/dsm/welcome-page';
+import { Study } from 'lib/component/dsm/navigation/enums/selectStudyNav.enum';
+import KitUploadPage from 'pages/dsm/kitUpload-page';
+import { SamplesMenu } from 'lib/component/dsm/navigation/enums/samplesNav.enum';
+import { KitType } from 'lib/component/dsm/samples/enums/kitType.enum'
+
+
+test.describe('RGP Kit Upload', () => {
+    let welcomePage: WelcomePage;
+    let homePage: HomePage;
+    let navigation: Navigation;
+
+    test.beforeEach(async ({ page }) => {
+        await login(page);
+        welcomePage = new WelcomePage(page);
+        homePage = new HomePage(page);
+        navigation = new Navigation(page);
+    });
+
+    test('Can upload a Blood & RNA kit @functional @rgp', async ({ page }) => {
+        //Go to DSM in order to do a kit upload (type: "BLOOD & RNA")
+        await welcomePage.selectStudy(Study.RGP);
```

Comment:
I think we want the assertions to live inside the tests, but use the page classes to return values.  So maybe have the home page return the study title or the welcome title, and then make the assertion in this file.